portfolio/src/main/webapp/LinkedIn_Info.html

```diff
@@ -0,0 +1,18 @@
+!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Web Information</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h2>LinkedIn</h2>
+      <p>
+           <a href="www.linkedin.com/in/charles-floeder-878a12192">LinkedIn URL</a>
```

Comment:
This line looks like it's indented a bit too much. Maybe remove two leading spaces?

Reply:
I just auto-formatted the doc now. Is there a style guide somewhere to know how you would like the formatting to be?

Reply:
Whatever the auto-formatter does is fine with me :)